### ISSUE TITLE
Incorrect error message for ADIOS2_USE_ENDIAN_REVERSE=ON

### ISSUE THREAD

**Issue by NAThompson** [ornladios/ADIOS2#1489]
The following error messages:

```
 ➜  ADIOS2 git:(master) ✗ grep -nr -B 2 -A 1 'ADIOS2_USE_ENDIAN_REVERSE=ON' .
./source/adios2/toolkit/format/bp4/BP4Deserializer.cpp-73-                "ERROR: reader found BigEndian bp file, "
./source/adios2/toolkit/format/bp4/BP4Deserializer.cpp-74-                "this version of ADIOS2 wasn't compiled "
./source/adios2/toolkit/format/bp4/BP4Deserializer.cpp:75:                "with the cmake flag -DADIOS2_USE_ENDIAN_REVERSE=ON "
./source/adios2/toolkit/format/bp4/BP4Deserializer.cpp-76-                "explicitly, in call to Open\n");
--
--
./source/adios2/toolkit/format/bp3/BP3Deserializer.cpp-83-                "ERROR: reader found BigEndian bp file, "
./source/adios2/toolkit/format/bp3/BP3Deserializer.cpp-84-                "this version of ADIOS2 wasn't compiled "
./source/adios2/toolkit/format/bp3/BP3Deserializer.cpp:85:                "with the cmake flag -DADIOS2_USE_ENDIAN_REVERSE=ON "
./source/adios2/toolkit/format/bp3/BP3Deserializer.cpp-86-                "explicitly, in call to Open\n");
```

suggests use of incorrect cmake flags:

```
$ cmake -DADIOS2_USE_ENDIAN_REVERSE=ON 
  Build Options:
      ...
      Endian_Reverse: OFF


CMake Warning:
  Manually-specified variables were not used by the project:

    ADIOS2_USE_ENDIAN_REVERSE
```

It seemed to me that a natural fix would be using `-DEndian_Reverse=ON`, but this didn't work either.

**Comment by williamfgc** [comment]
Hi @NAThompson , please use `-DADIOS2_USE_Endian_Reverse=ON` ( I got the name from boost BTW ). For more options see [the build section in readthedocs](https://adios2.readthedocs.io/en/latest/installation/installation.html#build-cmake-dvar-value-options)

**Comment by NAThompson** [comment]
Fixed in #1490.
